### PR TITLE
👷 Trigger nightly docs build on merge to `main`

### LINKED ISSUE #1: 👷 Trigger docs rebuild when `main` branch is changed
### Related problem

Once these tutorials are included in the user docs, Sphinx will need to run and translate the notebooks into ReStructuredText, then HTML each time this repository is changed. The logistics should be handled by CI.

### Proposed feature

GitHub Action to trigger a docs rebuild each time [the `main` branch of this repository](https://github.com/FCP-INDI/C-PAC_tutorials/tree/main) is updated

### Acceptance criteria

- [ ] Nightly user docs rebuild automatically whenever [the `main` branch of this repository](https://github.com/FCP-INDI/C-PAC_tutorials/tree/main) is updated

### Alternatives

_No response_

### Additional context

_No response_

### PR DESCRIPTION
**Pull request by shnizzedy**
## Fixes

Fixes #1 by @shnizzedy

## Description

Sets up a GitHub Action to trigger a rebuild of tutorials in the nightly user docs whenever [a pull request to the main branch of this repository](https://github.com/FCP-INDI/C-PAC_tutorials/pulls?q=is%3Apr+base%3Amain) is merged.

## Technical details

https://github.com/FCP-INDI/C-PAC_tutorials/blob/2851af78eaaa21cb368bd5d990c522042b0855c0/.github/workflows/rebuild-nightly-docs.yml#L3-L8
https://github.com/FCP-INDI/C-PAC_tutorials/blob/2851af78eaaa21cb368bd5d990c522042b0855c0/.github/workflows/rebuild-nightly-docs.yml#L12

https://github.com/FCP-INDI/fcp-indi.github.io/blob/58c501d509ea9c4898a129246a010b5922bbaa38/.circleci/config.yml#L112-L127

## Tests

https://github.com/FCP-INDI/C-PAC_tutorials/runs/6116077694?check_suite_focus=true → https://app.circleci.com/pipelines/github/FCP-INDI/fcp-indi.github.io/852/workflows/c1862d46-ae65-4345-81eb-d9d451988aee/jobs/378

## Checklist

- [x] My pull request has a descriptive title (not a vague title like `Update index.md`).
- [x] My pull request targets the `main` branch of the repository. 
- [x] My commit messages follow best practices.
- [x] My code follows the established code style of the repository.
- [ ] I added tests for the changes I made (if applicable).
- [ ] I updated the changelog.
- [ ] I added or updated documentation (if applicable).
- [x] I tried running the project locally and verified that there are no visible errors.

## Developer Certificate of Origin

<details>
<summary>Developer Certificate of Origin</summary>

```
Developer Certificate of Origin
Version 1.1

Copyright (C) 2004, 2006 The Linux Foundation and its contributors.
1 Letterman Drive
Suite D4700
San Francisco, CA, 94129

Everyone is permitted to copy and distribute verbatim copies of this
license document, but changing it is not allowed.


Developer's Certificate of Origin 1.1

By making a contribution to this project, I certify that:

(a) The contribution was created in whole or in part by me and I
    have the right to submit it under the open source license
    indicated in the file; or

(b) The contribution is based upon previous work that, to the best
    of my knowledge, is covered under an appropriate open source
    license and I have the right under that license to submit that
    work with modifications, whether created in whole or in part
    by me, under the same open source license (unless I am
    permitted to submit under a different license), as indicated
    in the file; or

(c) The contribution was provided directly to me by some other
    person who certified (a), (b) or (c) and I have not modified
    it.

(d) I understand and agree that this project and the contribution
    are public and that a record of the contribution (including all
    personal information I submit with it, including my sign-off) is
    maintained indefinitely and may be redistributed consistent with
    this project or the open source license(s) involved.
```

</details>

### CODE REVIEW

File: .github/workflows/rebuild-nightly-docs.yml [hunk 1]
```diff
@@ -0,0 +1,22 @@
+name: Rebuild nightly docs
+
+on:
+  pull_request:
+    types:
+      - closed
```
Comment (gkiar) [inline .github/workflows/rebuild-nightly-docs.yml:6]: Does this capture `merged` PRs, but exclude those that are `closed` without being `merged`?

**Review by gkiar — COMMENTED** [review]
While I can't vouch for your `shnizzedy/github-action-trigger-circleci@v1.1.1` GH action in this PR, it looks great! I had 1Q that I left

File: .github/workflows/rebuild-nightly-docs.yml (hunk 1 above)
Comment (shnizzedy) [reply .github/workflows/rebuild-nightly-docs.yml:6]: That + https://github.com/FCP-INDI/C-PAC_tutorials/blob/2851af78eaaa21cb368bd5d990c522042b0855c0/.github/workflows/rebuild-nightly-docs.yml#L12 does

**Comment by shnizzedy** [comment]
> While I can't vouch for your shnizzedy/github-action-trigger-circleci@v1.1.1 GH action

[That action](https://github.com/shnizzedy/github-action-trigger-circleci) is just a fork of https://github.com/marketplace/actions/github-action-trigger-circleci-job with https://github.com/zivkaziv/circleci-trigger-github-action/pull/4 and https://github.com/zivkaziv/circleci-trigger-github-action/pull/7 merged in, built and tagged

**Pull request merged**